services/ui/ws/platform_display_default.cc

```diff
@@ -318,7 +318,7 @@ void PlatformDisplayDefault::DispatchEvent(ui::Event* event) {
 }
 
 void PlatformDisplayDefault::OnCloseRequest() {
-#if !defined(USE_OZONE) || defined(CHROMEOS)
+#if defined(OS_CHROMEOS)
```

Comment:
I think there is some Windows configuration that might take this path.

Maybe we should flip the if/else logic:

#if defined(USE_OZONE) && defined(OS_LINUX) && !defined(OS_CHROMEOS)
  put here the code that is specific to us.
#else 
  put here the existing code
#endif 

wdyt?

Reply:
question: there is some more code that I remember that from display.cc we call window tree client directly.

any reason to change it here (as part of the fix)? or it is just to make things more uniform?

Reply:
Yes, it is here - https://github.com/Igalia/chromium/pull/231/commits/1206e59231c1e309b8090c223ec89a111e3f602a#diff-916ce6b8f717ba0d5120a6b052f1b5a6R384 . Part of that went to WindowTree now.

Reply:
I've changed the #if defined only because the previous author add #if defined here, which broke ChromeOS build (once you close the window, it crashes).